### PR TITLE
Validation page for events

### PR DESCRIPTION
**Pull request by benwerd**
# Description

Loads event, verifies signature, and displays a checkmark. This PR does not include any new designs. This will need to be heavily tested on staging.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: tickets/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "@types/react-redux": "7.0.8",
     "@types/storybook__react": "4.0.1",
     "@types/styled-components": "4.1.14",
-    "@unlock-protocol/unlock-js": "0.0.26",
+    "@unlock-protocol/unlock-js": "0.0.28",
```
Comment (julien51) [inline tickets/package.json]: Could you split that into its own pr please?

File: tickets/src/_server.js [hunk 2]
```diff
@@ -21,6 +21,11 @@ function _server(port, dev) {
           } else if (path === 'event') {
             const params = route('/event/:lockAddress')(pathname)
             app.render(req, res, '/event', Object.assign(params, query))
+          } else if (path === 'checkin') {
```
Comment (julien51) [inline tickets/src/_server.js]: make sure you add the corresponding changes in `_redirect` for Netlify?

File: tickets/src/components/content/EventVerify.js [hunk 3]
```diff
@@ -51,7 +67,13 @@ export const mapDispatchToProps = dispatch => ({
 })
 
 export const mapStateToProps = ({ router, locks, account, event }) => {
-  const { lockAddress } = lockRoute(router.location.pathname)
```
Comment (cellog) [inline tickets/src/components/content/EventVerify.js]: This is a rather brittle change, can we instead update the regexp and `lockRoute`?

File: tickets/src/__tests__/middlewares/walletMiddleware.test.js [hunk 4]
```diff
@@ -380,15 +380,16 @@ describe('Wallet middleware', () => {
       signedAddress: 'encrypted sig',
     }
 
-    mockWalletService.recoverAccountFromSignedData = jest.fn(
-      data => data.message.address.publicKey
+    mockWalletService.recoverAccountFromSignedData = jest.fn((data, sa, cb) =>
```
Comment (cellog) [inline tickets/src/__tests__/middlewares/walletMiddleware.test.js]: this is a good test. We also need a separate test that verifies that the real `walletService.recoverAccountFromSignedData` actually does what we expect it, in order to catch API regressions. Otherwise, should a change to `unlock-js` break this, we won't know until it's deployed to staging. The test would need to mock out the json-rpc call using `nock`, I can definitely help with this. We would borrow `nockHelper` from `unlock-js`

The test would also verify that our assumption that `recoverAccountFromSignedData` calls a callback is correct
